content/en/user-guide/polaris-catalog/index.md

```diff
@@ -0,0 +1,216 @@
+
+- Launching the Polaris Catalog service
+- Setting up an external volume
+- Creating a catalog integration
+- Creating an Iceberg table
+- Querying the Iceberg table
+
+### Start Polaris catalog container
+
+The following command starts the Polaris catalog container using the `localstack/polaris` Docker image:
+
+```bash
+docker run -d --name polaris-test \
+  -p 8181:8181 -p 8182:8182 \
+  -e AWS_REGION=us-east-1 \
+  -e AWS_ACCESS_KEY_ID=test \
+  -e AWS_SECRET_ACCESS_KEY=test \
+  -e AWS_ENDPOINT_URL=http://localhost:4566 \
+  -e POLARIS_BOOTSTRAP_CREDENTIALS=default-realm,root,s3cr3t \
+  -e polaris.realm-context.realms=default-realm \
+  -e quarkus.otel.sdk.disabled=true \
+  localstack/polaris:latest
+```
+
+Wait for Polaris to become healthy:
+
+```bash 
+curl -X GET http://localhost:8182/health
+```
+
+### Authenticate and create Polaris catalog
+
+Set variables and retrieve an access token:
+
+```bash
+REALM="default-realm"
+CLIENT_ID="root"
+CLIENT_SECRET="s3cr3t"
+BUCKET_NAME="test-bucket-$(openssl rand -hex 4)"
+CATALOG_NAME="polaris"
+
+TOKEN=$(curl -s -X POST http://localhost:8181/api/catalog/v1/oauth/tokens \
+  -H "Polaris-Realm: $REALM" \
+  -d "grant_type=client_credentials&client_id=$CLIENT_ID&client_secret=$CLIENT_SECRET&scope=PRINCIPAL_ROLE:ALL" | jq -r '.access_token')
+```
+
+The `TOKEN` variable will contain the access token.
+
+Create a catalog:
+
+```bash
+curl -s -X POST http://localhost:8181/api/management/v1/catalogs \
+  -H "Authorization: Bearer $TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "catalog": {
+      "name": "'"$CATALOG_NAME"'",
+      "type": "INTERNAL",
+      "properties": {
+        "default-base-location": "s3://'"$BUCKET_NAME"'/test"
+      },
+      "storageConfigInfo": {
+        "storageType": "S3_COMPATIBLE",
+        "allowedLocations": ["s3://'"$BUCKET_NAME"'/"],
+        "s3.roleArn": "arn:aws:iam::000000000000:role/'"$BUCKET_NAME"'",
+        "region": "us-east-1",
+        "s3.pathStyleAccess": true,
+        "s3.endpoint": "http://localhost:4566"
+      }
+    }
+  }'
+```
+
+Grant necessary permissions to the catalog:
+
+```bash
+curl -s -X PUT http://localhost:8181/api/management/v1/catalogs/polaris/catalog-roles/catalog_admin/grants \
+  -H "Authorization: Bearer $TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{"type": "catalog", "privilege": "TABLE_WRITE_DATA"}'
+```
+
+### Create a bucket
+
+Create a bucket using the `awslocal` command:
+
+```bash
+awslocal s3 mb s3://$BUCKET_NAME
+```
+
+### Create an external volume
+
+In your SQL client, create an external volume using the `CREATE EXTERNAL VOLUME` statement:
+
+```sql
+CREATE EXTERNAL VOLUME polaris_volume
+STORAGE_LOCATIONS = (
+  (
+    NAME = aws_s3_test
+    STORAGE_PROVIDER = S3
+    STORAGE_BASE_URL = 's3://test-bucket/'
+    STORAGE_AWS_ROLE_ARN = 'arn:aws:iam::000000000000:role/test-bucket-290de95d'
```

Comment:
replace `test-bucket` in two lines with `$BUCKET_NAME`

Reply:
This is going to be executed in a SQL client so need to hardcode the S3 bucket name and role.